.github/workflows/continous-integration.yml

```
@@ -88,89 +88,119 @@ jobs:
-    needs: [quality, test]     # only run after all other stages succeeded
+    needs: [quality, test] # only run after all other stages succeeded
 
     steps:
-    - name: Checkout git repository 🕝
-      uses: actions/checkout@v2
-
-    - name: Set up Python 3.6 🐍
-      uses: actions/setup-python@v1
-      with:
-        python-version: 3.6
-
-    - name: Read Poetry Version 🔢
-      run: |
-           echo "POETRY_VERSION=$(scripts/poetry-version.sh)" >> $GITHUB_ENV
-      shell: bash
-
-    - name: Install poetry 🦄
-      uses: Gr1N/setup-poetry@v4
-      with:
-        poetry-version: ${{ env.POETRY_VERSION }}
-
-    - name: Build ⚒️ Distributions
-      run: poetry build
-
-    - name: Publish to PyPI 📦
-      uses: pypa/gh-action-pypi-publish@37e305e7413032d8422456179fee28fac7d25187
-      with:
-        user: __token__
-        password: ${{ secrets.PYPI_TOKEN }}
-
-    - name: Publish Release Notes 🗞
-      if: env.IS_TAG_BUILD
-      env:
-        GITHUB_TAG: ${{ github.ref }}
-        GITHUB_REPO_SLUG: ${{ github.repository }}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: |
-        GITHUB_TAG=${GITHUB_TAG/refs\/tags\//}
-        sudo apt-get update
-        sudo apt-get -y install pandoc
-        pip install -U github3.py
-        python3 ${GITHUB_WORKSPACE}/scripts/publish_gh_release_notes.py
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.6 🐍
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+
+      - name: Read Poetry Version 🔢
+        run: |
+          echo "POETRY_VERSION=$(scripts/poetry-version.sh)" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Install poetry 🦄
+        uses: Gr1N/setup-poetry@v4
+        with:
+          poetry-version: ${{ env.POETRY_VERSION }}
+
+      - name: Build ⚒️ Distributions
+        run: poetry build
+
+      - name: Publish to PyPI 📦
+        uses: pypa/gh-action-pypi-publish@37e305e7413032d8422456179fee28fac7d25187
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_TOKEN }}
+
+      - name: Publish Release Notes 🗞
+        if: env.IS_TAG_BUILD
+        env:
+          GITHUB_TAG: ${{ github.ref }}
+          GITHUB_REPO_SLUG: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          GITHUB_TAG=${GITHUB_TAG/refs\/tags\//}
+          sudo apt-get update
+          sudo apt-get -y install pandoc
+          pip install -U github3.py
+          python3 ${GITHUB_WORKSPACE}/scripts/publish_gh_release_notes.py
+
+  send_slack_notification_for_release:
+    name: Notify Slack
+    runs-on: ubuntu-latest
+    if: always()
+    needs:
+      - deploy
+
+    steps:
+      - name: Get RASA SDK Version
+        env:
+          RASA_SDK_VERSION: ${{ github.ref }}
+        run: |
+          RASA_SDK_VERSION=${RASA_SDK_VERSION/refs\/tags\//}
```

You have to set this variable as an environment variable.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
```suggestion
          echo "RASA_SDK_VERSION=${RASA_SDK_VERSION/refs\/tags\//}" >> $GITHUB_ENV
```